site/ts/activity-dashboard.js

```diff
@@ -209,9 +220,11 @@ export function columnOnClick(n) {
     }
 
     for (i = 0; i < 10; i++) {
+        // eslint-disable-next-line eqeqeq
         if (i != n && $(`#${i}`).children('i').hasClass('fa-angle-up')) {
```

Comment:
All usages of this function pass `n` as a number.

```suggestion
        if (i !== n && $(`#${i}`).children('i').hasClass('fa-angle-up')) {
```